src/features/metadata-instance-editor/Instance.js

```diff
@@ -72,6 +74,7 @@ type State = {
     data: Object,
     errors: { [string]: React.Node },
     isAIFolderExtractionEnabled: boolean,
+    cascadePolicyConfiguration?: MetadataCascadePolicyConfiguration | null,
```

Comment:
i don't think it needs the optional?
```suggestion
    cascadePolicyConfiguration: MetadataCascadePolicyConfiguration | null,
```

Reply:
Without the null, flow didn't like it.